### PR TITLE
Add uxarray package

### PR DESCRIPTION
**Pull request by michaelavs**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by michaelavs** [comment]
@conda-forge/help-python This recipe is ready for review!

File: recipes/uxarray/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/UXARRAY/uxarray/archive/v{{ version }}.tar.gz   # For building from a specific tag's tarball
+  sha256: 3cb04f27168ac946ae189fef55f95e19a4b6248c9af847d8ed5c699d3c24b6c1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36]
```
Comment (dopplershift) [inline recipes/uxarray/meta.yaml:14]: This looks like it's pure Python, so could use noarch instead and drastically cut down the number of builds:
```suggestion
  noarch: python
```

File: recipes/uxarray/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/uxarray/meta.yaml:22]: For noarch:
```suggestion
    - python >=3.6
    - pip
    - setuptools
  run:
    - python >=3.6
```

**Review by dopplershift — COMMENTED** [review]
It would be good to make this noarch if possible. Also, I need confirmation from  @paullric, @erogluorhan, @anissa111, @rajeeja, @chengzhuzhang, and @rljacob that they are willing to be listed as maintainers for this recipe.

**Comment by michaelavs** [comment]
> It would be good to make this noarch if possible. Also, I need confirmation from @paullric, @erogluorhan, @anissa111, @rajeeja, @chengzhuzhang, and @rljacob that they are willing to be listed as maintainers for this recipe.

I forgot to push changes for the maintainer list last week! The only change that I am aware of would be removing @rljacob from the list (which I just committed the edit for)

**Comment by anissa111** [comment]
@dopplershift I am willing to be listed as a maintainer.

**Comment by chengzhuzhang** [comment]
@dopplershift I'm willing to be listed as a maintainer.

**Comment by erogluorhan** [comment]
@dopplershift I am willing to be listed as a maintainer.

**Comment by paullric** [comment]
@dopplershift I am willing to be listed as a maintainer.

**Comment by rajeeja** [comment]
I am willing to be listed as a maintainer.

**Pull request merged**